### PR TITLE
docs(datetime): add button playgrounds

### PR DESCRIPTION
**Pull request by liamdebeasi**
This PR adds the following playgrounds:

1. Showing default confirmation buttons.
2. Customizing default confirmation button texts.
3. Slotting in custom confirmation buttons.

These fall under the "Buttons" category.

### CODE REVIEW

File: static/usage/datetime/buttons/customizing-buttons/javascript.md
Comment (sean-perkins) [inline static/usage/datetime/buttons/customizing-buttons/javascript.md:11]:
```suggestion
  var datetime = document.querySelector('ion-datetime');
```

Stackblitz limitation... if you click any of the buttons in your live example you will get an exception. For whatever reason they don't allow `const` or `let` at the document scope.

**Review by sean-perkins — CHANGES_REQUESTED** [review]

File: docs/api/datetime.md
Comment (amandaejohnston) [inline docs/api/datetime.md:232]: I'm not seeing the footer/buttons on any of the datetimes when in `md` mode, although they do show in the Stackblitz so I assume they're supposed to be there 🤔

**Review by amandaejohnston — APPROVED** [review]

**Review by sean-perkins — APPROVED** [review]
Good to merge once merge conflicts are resolved.

**Pull request merged**
